server.js

```diff
@@ -711,6 +711,19 @@ const envVarsCheck = callback => {
   }
 };
 
+const couchDbNoAdminPartyModeCheck = callback => {
+  const noAuthUrl = process.env.COUCH_URL.replace(/\/\/[^@]+@/, '//'),
+        protocol = noAuthUrl.split('://')[0];
```

Comment:
I was originally like this (well, the real code is grosser than this because the pre 8.x url api is not great), but then I got rid of it for some reason. I can't remember why though. I'll change it back and see if whatever issue I had before comes up.